addons/website_gengo/static/src/css/website_gengo.css

```diff
@@ -1,3 +1,3 @@
-.o_translatable_inprogress {
-  background: #b7e4ff;
-}
\ No newline at end of file
+html[lang] > body.editor_enable [data-oe-translation-state="inprogress"] {
```

Comment:
do not try to override css property using important.

you should add o_translatable_inporgress class when changed oe_translation-state data value.